### ISSUE TITLE
Notification stretches full-width

### ISSUE THREAD

**Issue by techanvil** [google/site-kit-wp#9599]
## Feature Description

This is largely due to the dashboard and settings areas not being constrained to a max-width (which we should do and I believe was planned at some point but we haven't gotten to it).  We should probably solve this on the larger level rather than trying to address it on individual components like this, but it looks odd to have components stretch so wide, especially those with a small amount of content. @sigal-teller 

Similar notifications in module setup only stretch ~980 to 1024px IIRC

As raised by @aaemnnosttv, see the related [Asana task](https://app.asana.com/0/1208235052195562/1208490790420488).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* [ ] 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by techanvil** [comment]
It's worth noting we've introduced a similar notification for Key Metrics:

![Image](https://github.com/user-attachments/assets/8e209bd2-52fd-4384-9c73-b6db94028e67)

**Comment by techanvil** [comment]
Hi @sigal-teller, as @aaemnnosttv has pointed out and can be seen in the screenshots above, the new notifications can look a bit odd in wider viewports.

We do constrain the width of existing/old-style notices for example here in the AdSense setup flow:

![Image](https://github.com/user-attachments/assets/251f04cf-857f-466e-a017-647eb6d682a1)

And, here's the same component being used with a CTA on the dashboard (this one looks a bit odd anyway in such a wide viewport):

![Image](https://github.com/user-attachments/assets/8b32df7d-622c-4b6b-b61b-025408da4916)

As Evan has mentioned, it would be good to address this sort of thing in a holistic manner and adding a maximum width to the entire page would be well worth exploring. In fact it would be very easy to implement and I've mocked it up for illustration. I did a quick search for common maximum page widths and used this top-hit [article](https://www.katescott.co/blog/squarespace-responsive-web-design) as a reference.

Here's how the dashboard could look width a 1920px max-width:

![Image](https://github.com/user-attachments/assets/d73e4456-a09d-4f94-97df-629b21bbb882)

![Image](https://github.com/user-attachments/assets/99cf2238-104c-45a3-87e1-a511c67a7968)

And here's the Settings screen:

![Image](https://github.com/user-attachments/assets/8d3b31bd-3b4d-4f75-a6f8-71dbf06e54c3)

![Image](https://github.com/user-attachments/assets/2df32e73-5987-4068-90fa-7922e8ebe0e6)

By comparison here's the dashboard at 1280px (I'd say 1920px is better):

![Image](https://github.com/user-attachments/assets/0fa0d0dd-dcc4-4540-a431-76067d501db7)

![Image](https://github.com/user-attachments/assets/fcdb24cd-26f4-4f92-abb2-ee4eba40306a)

And for reference, here are both the dashboard and Settings screen at the current unconstrained width:

![Image](https://github.com/user-attachments/assets/a1925ed7-ec12-40af-817e-2fc2da762f20)

![Image](https://github.com/user-attachments/assets/950d3fa8-eb04-4200-8486-55f6d857cff8)

![Image](https://github.com/user-attachments/assets/798180bc-a6ca-4c2a-a33b-6a6e3baea04f)

I'm interested to get your thoughts on this, do you think we should add a maximum width to the pages as above and/or consider a d more granular approach?

**Comment by sigal-teller** [comment]
Hi @techanvil Thank you for all your work here, it's very helpful. Yes, I think we should add a maximum width limit. In wide viewports the content stretches in a way that sometimes the CTA in the right is placed so far from the content and I'm not it can be missed. I think that we can improve the view by adding the navigation to the left of the content, like a vertical bar or something but that will require some effort both on the design side and the engineering side. I know that it was previously planned at some point but it was paused. 

I think that for now we can settle for a simpler solution which will be an improvement to what we currently have. What you suggested sounds great. However I'm not about the center alignment since in the page there is a left bar (the WP bar). We can also align everything to the left, but that means we will need to align the nav bar as well. 

I'm adding and example here of how it could look. I created a mock for wider viewport and a narrower one, just so it will be clear. I would love to hear your thoughts on this.

![Image](https://github.com/user-attachments/assets/67cd43d8-2054-437a-8f90-9a4f5fcd53c3)
![Image](https://github.com/user-attachments/assets/7b3bbb21-63a7-4d16-8785-87723feb2b5f)

**Comment by techanvil** [comment]
Thanks @sigal-teller!

Your point about the admin menu is a good one, and the suggested layout is logical. However, to be totally honest I must admit I find it a little odd. To try to analyse my reaction, I suppose that with a wide screen the page feels a bit unbalanced, and aligning the nav to the left makes it a bit less prominent.

I thought I'd try retaining the centre-alignment for the nav but constrained to the max width, but I'm not sure it improves it:

![Image](https://github.com/user-attachments/assets/420aa593-04e9-4028-aba4-83245810b96c)

Coming back to your proposed layout; I am sure I'd get used to it, and it's only relevant for very wide viewports, so it's unlikely I'd see it that often anyway. Nevertheless, it does seem a bit unusual and it's hard to find examples of high traffic sites which have a similar layout, all of the ones I've tried have either a centred layout or unconstrained width. I feel like the left-aligned max width layout is something I've seen more of a few years ago, although there are bound to be plenty of current examples if you know where to look.

Ultimately though, if we want to add a maximum width to the main content then it does seem a better fit when considering the position of the left hand menu. We _could_ move the menu itself, adding a left-hand margin to bring the menu to the centred content but I don't seriously think we should, it would look odd switching between the other admin pages if we did that and the menu isn't really styled well for it either.

One thing I noticed while considering this is that the rest of the WordPress admin interface doesn't have a max width, it's unconstrained across all the standard pages, for example:

![Image](https://github.com/user-attachments/assets/3f18089b-6cc5-4b8f-9a17-53e489cb2323)

However, it's clear Site Kit would benefit from a constraint to the maximum width, so unless we have other alternatives to consider I'd be happy enough taking the left-aligned approach. Sorry, I know it's not the most wholehearted endorsement but that's my feedback, hope that it's helpful and interested to see what you think.

**Comment by techanvil** [comment]
Hi @sigal-teller, sorry for the delay replying, I had to prioritise FPM issues, and let this one slip a bit. I hear you on the other things on the plate front!

It strikes me this is such a substantial change to the layout, and without an obvious choice to make, it's not something we need to rush into and would benefit from taking a more considered approach with some additional research as time allows.

I'd also be interested in getting @aaemnnosttv's thoughts, as this stems from an issue he raised during the Audience Segmentation bug bash. We should of course also run this past @marrrmarrr to get her input and approval. I'll tag Evan on Slack in the first instance and we can take it from there.

**Comment by aaemnnosttv** [comment]
Thanks for all the thought you've all put into this so far – my thoughts are as follows:

I think we need not look much farther than the Google Analytics front-end "Home" and Search Console dashboard views as the closest parallels to the SK dashboard for guidance that is reasonable to follow (perhaps with more weight on GA since that's where most of the dashboard data comes from.

These interfaces share these common traits:

- Full-width header
- Centered max-width content that does not grow at larger screen resolutions

GA
<img width="1473" alt="Image" src="https://github.com/user-attachments/assets/0e19390f-0b47-4082-bef0-f500069b5e04" />

SC
<img width="633" alt="Image" src="https://github.com/user-attachments/assets/4e438044-f705-4ea7-9c42-fe11eec2351e" />

GA has a content width of **1280**px where SC has only **912**

<img width="1343" alt="Image" src="https://github.com/user-attachments/assets/5f9f7dff-9d08-4d2c-bce0-8a30bc82f730" />

<img width="937" alt="Image" src="https://github.com/user-attachments/assets/d0a4f9f5-5a28-4a53-a1bd-2f2df11188e7" />

---

In my experience, using a 1920w viewport, having things full-width is too wide (which is where this conversation started a while back). On the dashboard currently when the admin menu is open, the dashboard's content stretches to **1677**px

<img width="1766" alt="Image" src="https://github.com/user-attachments/assets/dc13bc3b-61bf-48f8-b3ca-4297c770d557" />

IMO, if we matched the content width of GA (also centered), it looks much more comfortable. Matching SC would be too narrow I think.

<img width="1759" alt="Image" src="https://github.com/user-attachments/assets/b7647f7d-715e-4fd2-9d2b-355b6f0d7012" />

It's worth noting that I think we should keep a max-width on text elements, such as text in notifications (as we do now?) to maintain good readability (see example from GA below). Together with a limited max width, I think we can have a more cohesive experience.

<img width="1512" alt="Image" src="https://github.com/user-attachments/assets/ac6e1b43-2eec-4d0d-a76a-6629dd7d6481" />

**Comment by sigal-teller** [comment]
@aaemnnosttv SGTM. I think that we can use this approach. There are still some elements we'll need to define, like the behavior of the lower right corner floating banners that we started to use, which might feel detached when placed outside the content area.

**Comment by techanvil** [comment]
Thanks @aaemnnosttv & @sigal-teller. Using the Analytics home page as a reference is a good shout.

My only concern is how a 1280px max width would impact the Key Metrics and Audience tiles.

For Key Metrics, there's more chance of content being truncated, and it would limit our ability to add other view types e.g. graphs in the tiles.

![Image](https://github.com/user-attachments/assets/57dafabd-03a8-4ad8-b756-6f4888e3c475)

For the Audience tiles, it prevents the ability to show the full three tile view without truncating one of them:

![Image](https://github.com/user-attachments/assets/0debe759-6da6-4983-a9b1-70534d30312b)

I would suggest we choose a common viewport width that's a little higher than 1280px, e.g. 1366 or 1440. Here's how the Audience tiles would look:

1336px:
![Image](https://github.com/user-attachments/assets/aaff28fa-d4d3-43a5-b9b7-87c8d1dc09a1)

1440px:
![Image](https://github.com/user-attachments/assets/19559735-b71f-45ff-9f5e-aea368640683)

WDYT?

**Comment by sigal-teller** [comment]
Thank you @techanvil I think that 1366px can work well as it's pretty close to the original designs width, if we're taking into account the additional width of the WP nav bar.

**Comment by techanvil** [comment]
Thanks @sigal-teller, that SGTM.

What do you think about the next steps here? You made a good point that we need to consider some other elements e.g. the lower right corner floating banners. It seems to me these should be addressed at the same time that we add the max width, whereas it might be sensible to address the maximum width text elements that @aaemnnosttv mentioned separately so as to allow us to take an iterative approach. Or, do you think we should prepare the design for all of these aspects up front so we can consider it holistically? Also, are there any other elements that spring to mind that we should address?

**Comment by sigal-teller** [comment]
Thanks @techanvil I'm not sure how this should be handled in terms of splitting this into several tasks, I guess this depends on engineering considerations as well. As for the elements that we need to take into account, I can't think of any other floating components besides the introduction banners, which we'll need to decide whether we should place them relative to the content or page boundaries. We also have the call out for banners dismissal that point on the settings in the left nav bar but shouls remain pointing at the right place so it won't change.
Can you think of anything else?

**Comment by techanvil** [comment]
Thanks @sigal-teller. Splitting the development into multiple issues is indeed more of an engineering concern. I was more thinking about things from a design perspective, whether some aspects might impact others when considering the changes as a whole, implying we should get the full design ready before starting engineering, or if we're comfortable defining and building it in iterative cycles.

On a related note, we should consider how we want to release the changes - if we'd be happy to for it to be potentially iterated on over multiple releases or if it should be released in one go (implying it should be behind a feature flag until it's ready for release).

Realistically, it's not going to be a huge body of work, so it shouldn't take too long to implement and can probably be tackled in a single sprint anyway, but we should allow for it being worked on over multiple sprints to be on the safe side; even if it's a relatively small amount of work, other deliverables may be prioritised over it.

I've taken a look through the plugin for components that we should consider when making the changes and have identified the following.

Notifications in the header:
![Image](https://github.com/user-attachments/assets/5944794e-3e80-428a-b154-89f9249df865)

![Image](https://github.com/user-attachments/assets/84ca349c-1029-4aca-b186-4c647fe9a099)

Alignment of selection panels:
![Image](https://github.com/user-attachments/assets/6be3d05b-e9dd-432b-abb1-1114a500dc7f)

![Image](https://github.com/user-attachments/assets/cc6e7972-0a79-4d2f-ac68-a50082c9d4dc)

Footer navigation in the questionnaire screens:
![Image](https://github.com/user-attachments/assets/b989aef0-cd9d-4b62-b698-a9f28fe3745f)

Overlay notification components and surveys:
![Image](https://github.com/user-attachments/assets/456ecb2c-55ef-4361-839e-6c95ddad4f9f)

![Image](https://github.com/user-attachments/assets/4baf923d-aa7c-4966-b1cc-834b6787b5c4)

![Image](https://github.com/user-attachments/assets/3b57a090-5f09-4a80-a167-59df779be645)

**Comment by jimmymadon** [comment]
@aaemnnosttv @techanvil When discussing the redesign of our Banner Notifications, @sigal-teller mentioned this issue. The newly designed Banner Notifications have been re-designed in a way that assumes that we will have a max-width for Site Kit content which is the aim of this issue. So ideally, this issue / set of issues need to be prioritised and worked on sooner than later. 

What are the next steps here in terms of finalising the ACs / updating the scope and issue title here?

c.c. @binnieshah @ivonac4

**Comment by sigal-teller** [comment]
Hi @techanvil I think that the only component that requires attention would be the overlay banners/survey. I did address it in the banner notification epic while addressing this type of banner appearance in wider viewports. You can see it [here](https://www.figma.com/design/Jwd4oWQPtTJ4W9Y6JwMJjJ/Notifications?node-id=1398-12876&t=wWxXbkyDRzOeUUZu-1). So basically we should place it within the content area to avoid detached placement in cases of extremely wide viewports.
Is there anything else that requires definition that is blocking from moving forward with this issue?

**Comment by techanvil** [comment]
Thanks @sigal-teller.

So, just to be clear:

- We can leave the selection panels at the right-hand side of the screen, and the footer navigation in the questionnaire screens can be full-width.
- We should however update the placement of the overlay banners & surveys as you've described.

How about the notifications in the header that I've identified in the screenshots above - are they taken care of in the banner notification designs? If so, how should we manage them in conjunction with the other maximum width work - it could look a bit jarring if we release the initial max-width changes without addressing the notifications, so maybe it needs to stay behind a feature flag until the notifications are ready too. WDYT?

**Comment by jimmymadon** [comment]
@techanvil How about the notifications in the header that I've identified in the screenshots above - are they taken care of in the banner notification designs?

Yes, these are going to be redesigned as part of BNR3. The [Setup CTAs](https://www.figma.com/design/Jwd4oWQPtTJ4W9Y6JwMJjJ/Notifications?node-id=1147-46327&t=up1scu7G9lMwRVRr-0) will follow the pattern of widgets (only stretch to the content width and will have a margin all around them with rounder corners). As for the [remaining Banner Notifications](https://www.figma.com/design/Jwd4oWQPtTJ4W9Y6JwMJjJ/Notifications?node-id=1170-5013&t=up1scu7G9lMwRVRr-0), their content will fit to the max content width but their background colour will stretch to the full width of the viewport.

> If so, how should we manage them in conjunction with the other maximum width work - it could look a bit jarring if we release the initial max-width changes without addressing the notifications

I think the ACs for these issues here will be to do the bare minimum so that the white background in the Banner in your screenshot stretches the full-width of the viewport with everything else remaining the same.

**Comment by techanvil** [comment]
Thanks @jimmymadon - that sounds good.

I've updated the AC and moved the issue to IB. This will give us a good foundation to work on for BNR3 and also further refining maximum width content (e.g. text content that would benefit from a max width to make it easier to read).

**Comment by benbowler** [comment]
One addition not covered by the AC which could be is the module setup flow which still shows full width in this PR. Let me know if we should add this to the AC and IB.

**Comment by techanvil** [comment]
Hi @benbowler, thanks for flagging this. _Technically_ the module setup flow is rendered in the dashboard screen from a WP perspective, but this isn't clear from the AC, which I've updated.

I have also added a mention of the splash screen, which was also overlooked.

**Comment by benbowler** [comment]
Adding max-width to the class `googlesitekit-setup` applies this limit to both. Updated the IB.

**Comment by techanvil** [comment]
Thanks @benbowler! It's looking good, please note I've added a couple of comments to the PR and tweaked the IB to refer to one of them.

However, I've also noticed a couple of more things that need addressing:
- The entity dashboard will need a similar amendment as the main dashboard.
- The `OfflineNotification` is visually a sort of overlay notification and should be aligned similarly to the others.

**Comment by techanvil** [comment]
Thanks for updating the IB, @benbowler. A couple of minor tweaks are needed and it should be good to go.

- The `ConsentModeSetupCTAWidget` component _should_ be wrapped in the `.googlesitekit-page-content` div for the entity dashboard, we don't want to show [this](https://google.github.io/site-kit-wp/storybook/develop/?path=/story/consent-mode-consentmodesetupctawidget--default) full-width. Actually it looks like this instance of the setup CTA widget was missed in the recent Banner Notifications refactor, let's check in with @jimmymadon on this.
- It would be worth specifying the changes for `assets/sass/components/settings/_googlesitekit-settings-notice.scss` should be applied to the `.googlesitekit-settings-notice-offline-notice` class. This is already there in the PR, but the IB would benefit from the clarity.

**Comment by jimmymadon** [comment]
> Actually it looks like this instance of the setup CTA widget was missed in the recent Banner Notifications refactor

That's correct! We missed the <ConsentModeSetupCTAWidget> on the Entity Dashboard (we did refactor the one on the Main Dashboard). I'll do this as part of my current PR that refactors another notification. It might be a stretch to create a whole new issue for this. So for now, this is fine.

On a separate note, I can see that we are wrapping the entire `<Notifications>` component. Bear in mind that "Banners" (i.e. all large notifications in our plugin currently other than Setup CTA Widgets) should stretch to full width when it comes to their background colour (currently "white" as there is no background set but in the future, they will have green, yellow, red background colours as per their new Figmas), even though their content should be constrained to the max-width set here.

**Comment by benbowler** [comment]
@techanvil updated.

@jimmymadon all banners are above the Nav right now right? So the notifications area that will be within the max-width column will be the `NOTIFICATION_AREAS.BANNERS_BELOW_NAV`. Are there any of this area that should be full width white?

**Comment by techanvil** [comment]
Thanks @jimmymadon and @benbowler!

To clarify the point about the notifications, it's actually just those with the area slug `NOTIFICATION_AREAS.BANNERS_BELOW_NAV` with group ID `NOTIFICATION_GROUPS.SETUP_CTAS` that are rendered by the `Notifications` component within the max-width div. So, these ones should be OK as specced, AFAICT.

That said, @jimmymadon I've realised that I misread this point of yours in an earlier [comment](https://github.com/google/site-kit-wp/issues/9599#issuecomment-2670742692):

> I think the ACs for these issues here will be to do the bare minimum so that the white background in the Banner in your screenshot stretches the full-width of the viewport with everything else remaining the same.

I now realise that you were pointing out how the banners within the header will have their content constrained to the maximum width, but their background will be full width. Misreading this, I specified that these banners should simply be full width in the AC.

I do agree with your suggested approach, it's a better UX which is more aligned with the forthcoming designs.

@benbowler, apologies for another AC update, but I've amended it accordingly, please can you update the IB? I think it should be a case of adding a suitable CSS class to `SubtleNotifications`, and adding a max-width to `googlesitekit-subheader`, but there could be more to it, or a better solution.

**Comment by techanvil** [comment]
Thinking about `SubtleNotifications`, as they are only rendered on the dashboard, maybe we should move them out of `Header` and into `DashboardMainApp` / `DashboardEntityApp` within the max-width div... 🤔

**Comment by jimmymadon** [comment]
@benbowler 

> all banners are above the Nav right now right? So the notifications area that will be within the max-width column will be the NOTIFICATION_AREAS.BANNERS_BELOW_NAV. Are there any of this area that should be full width white?

Thanks for raising this. I think I missed this change in my engineering kick off - but as part of BNR3, ALL banners and notifications will ALWAYS be below the Nav Bar. The only difference is that all "Banners" ([like this one](https://www.figma.com/design/Jwd4oWQPtTJ4W9Y6JwMJjJ/Notifications?node-id=1170-5013&t=tFWUKgkbRkbL8EEA-0)) will continue to "stick" below the Nav Bar and have their background stretched from side to side while the content is still max-width constrained and centred. The "Setup CTA Widgets" and "Notices" (Subtle Notifications) will behave the way they do now, i.e. with a gap between the Nav bar header and their backgrounds won't be stretched.

I was originally going to move every BNR3 designed component into `NOTIFICATION_AREAS.BANNERS_BELOW_NAV` and then completely get rid of the `NOTIFICATION_AREAS.BANNERS_ABOVE_NAV` area. However, if required we can still have 2 areas, one for the ones that "stick" to the header and others that don't. We can rename the constants appropriately then.

**Comment by benbowler** [comment]
@jimmymadon @techanvil I think we should have distinct areas because adding the max-width within the `SubtleNotification` could cause unexpected issues if this notification type is used in other notification areas in future.

> **Banner Notifications**
> 
>  * [ ] Create two new notifications groups with the existing `BANNERS_BELOW_NAV` section:
>    * `NOTIFICATION_GROUPS.DASHBOARD_INLINE` - For all SubtleNotifications and other notifications that appear within the page max-width column.
>    * `NOTIFICATION_GROUPS.DASHBOARD_STICKY` (Rename `SETUP_CTA`) - For all notifications (Such as [Setup CTAs](https://www.figma.com/design/Jwd4oWQPtTJ4W9Y6JwMJjJ/Notifications?node-id=1170-5013&t=tFWUKgkbRkbL8EEA-0) that should handle their own max-width and sticky positioning in relation to the header.

I've added a new section to set out an approach, as some of this is related to BNR3, do you think some of this work should be split into a blocking ticket in the BNR epic rather than this ticket? Or perhaps we move this issue into the BNR epic so we can track it and make sure as notifications are refactored they are added to the appropriate notification area.

**Comment by jimmymadon** [comment]
@benbowler 

Just to clarify, NOTIFICATION_GROUPS are a different concept to NOTIFICATION_AREAS. Groups allow us to create separate queues that are computed and populated separately to speed up computing notifications in each queue. Areas allow us to display notifications from one or many groups in a designated area on the screen. 

>I've added a new section to set out an approach, this is related to BNR3, do you think some of this work should be split into a blocking ticket in the BNR epic

I'll dig into this more later today and see what are the possible options here. Leave this section of the IB with me. Thanks.

**Comment by benbowler** [comment]
Thanks @jimmymadon

**Comment by jimmymadon** [comment]
@benbowler  @binnieshah 

I have added #10806 as a blocker to this issue. However, since that one already has an IB, I've added the `Next Up` label to prioritise this in the next sprint as this issue adds a huge benefit to the design of our dashboard.

However, as I originally thought, this issue is not going to be a blocker to the remaining BNR3 "Banner" issues (10384/385/386/388).

**Comment by eugene-manuilov** [comment]
IB ✔

**Comment by benbowler** [comment]
While working on this also note the progress bar added for the loading state within the `SetupCTA` component in #10813, this will need to be considered as it should be full width with the header potentially escaping the max-width column:

https://github.com/google/site-kit-wp/blob/288a008ddb0fcc29c61c9d9046ccff5c20b11107/assets/js/googlesitekit/notifications/components/layout/SetupCTA.js#L95-L100

**Comment by benbowler** [comment]
Another note for the engineer that executes on this ticket, the grid wrapping the current notification area on the dashboard for SetupCTA's, Warning's etc adds additional margin above and below the component which should be handled within the component otherwise the component cannot control it's own layout. Eg. the Zero data banner currently has extra padding vs the [Figma mock](https://www.figma.com/design/Jwd4oWQPtTJ4W9Y6JwMJjJ/Notifications?node-id=1299-19360&t=yf5BSu4WIdgr04hL-0):

<img width="1104" alt="Image" src="https://github.com/user-attachments/assets/330248fc-54ef-4aa3-844f-d4517400cbf5" />

![Image](https://github.com/user-attachments/assets/33f74edc-1b60-43f9-a95a-1e957153edeb)

**Comment by jimmymadon** [comment]
@mohitwp I have brought this back into Execution because the newly designed Banner Notifications within the Header area are not aligned correctly as they should.

<img width="2117" height="943" alt="Image" src="https://github.com/user-attachments/assets/5ec97c26-0a40-487e-b794-5cca2ff1b0ed" />

I have already created a follow up PR, but need to clear out a couple of inconsistencies regarding padding within our plugin with Sigal. 

c.c. @10upsimon

**Comment by mohitwp** [comment]
### QA Update ⚠️ 

- Tested on dev environment.
- Tested Splash screen, settings, main dashboard, entity dashboard and view only dashboard.
- Tested offline notice, overlay notification and surveys.
- Verified The splash, dashboard (including module setup flows), and settings screens should have a maximum width of 1366px for the main content area (that is to say, Site Kit's content excluding headers and footers).
- Verified The max-width content centered horizontally within the plugin's content area.
- Verified Setup CTA widgets and Notice type notifications (originally known as subtle notifications) constrained to the max-width as per this [Figma](https://www.figma.com/design/Jwd4oWQPtTJ4W9Y6JwMJjJ/Notifications?node-id=1150-49611&t=4LF9esNwXXeWauaB-0).

@jimmymadon @10upsimon Please find a couple of my observations below


**_Issue 1:_** The overlay notification is not aligned with the dashboard content — it appears outside the content width of 1366px. 

<details>

https://github.com/user-attachments/assets/f7151b0f-c9fd-4523-8278-336b5edb7aae

<img width="3380" height="1456" alt="Image" src="https://github.com/user-attachments/assets/c23c96f0-4b62-467a-a7b9-47115706e574" />

<img width="3344" height="1332" alt="Image" src="https://github.com/user-attachments/assets/cefd4788-a6e8-4860-94ca-9a8bf8ac121f" />

</details>

**_Issue 2:_**  The success notice that appears after completing the SK setup (without Analytics) has its content and background color constrained to a maximum width of 1366px.
This is inconsistent with other notices like “Zero data” and “Gathering data”, which have a full-width background across the desktop screen.

<details>


<img width="3274" height="1282" alt="Image" src="https://github.com/user-attachments/assets/01f17385-dcfe-40e8-bf14-17635f323635" />

<img width="3188" height="1242" alt="Image" src="https://github.com/user-attachments/assets/e84dcbc7-cc48-4cec-b336-1a585238f93c" />

</details>

**PASS CASES**

<details>

https://github.com/user-attachments/assets/573ae29a-d2f6-49f3-82bf-b3070c0c9e09

https://github.com/user-attachments/assets/2ad85b8e-be04-4250-9332-17b07ab972ea

https://github.com/user-attachments/assets/7e6f0d93-9fc6-467e-aa19-8c869d33b5ad

https://github.com/user-attachments/assets/e2d34ba0-cdef-498e-821f-9384039f8be1

<img width="3204" height="1278" alt="Image" src="https://github.com/user-attachments/assets/b5f2bb16-dec8-4c1a-b7c9-1d9f1557a69a" />

<img width="1917" height="862" alt="Image" src="https://github.com/user-attachments/assets/0da9e8b9-fd2a-4663-8e53-649e71638b26" />

<img width="3180" height="1274" alt="Image" src="https://github.com/user-attachments/assets/28e57e4c-2bd4-4aed-8de3-6a5987efec24" />

<img width="3214" height="1298" alt="Image" src="https://github.com/user-attachments/assets/4fe3dcf2-1b22-472f-9fb5-fe8f6501a97c" />

<img width="3180" height="1264" alt="Image" src="https://github.com/user-attachments/assets/4328a1d9-39af-49e5-ac71-c0efc1b31170" />

<img width="3212" height="1498" alt="Image" src="https://github.com/user-attachments/assets/22cb182a-7d07-4751-8a73-c4025efdfad5" />


<img width="3206" height="1256" alt="Image" src="https://github.com/user-attachments/assets/2010e6e6-3e1a-41eb-b6cd-3460faf6c5ea" />

<img width="1699" height="731" alt="Image" src="https://github.com/user-attachments/assets/efe8ba06-aaed-4bdc-be83-6bff515d393b" />

</details>

**Comment by mohitwp** [comment]
@jimmymadon This banner from server side also require a fix. 

<img width="1919" height="525" alt="Image" src="https://github.com/user-attachments/assets/11cc97ee-791c-415f-b1d2-1f6ccc3e55e8" />

**Comment by jimmymadon** [comment]
@mohitwp The follow up PR fixes all of the issues you have reported above:
- the Overlay Notification alignment on all viewports and both, when WP Admin Menu is open and closed.
- the Success Notice Banner is now full width. PLEASE NOTE that the SVG image had to be changed here so it now uses [this](https://www.figma.com/design/Jwd4oWQPtTJ4W9Y6JwMJjJ/Notifications?node-id=1843-53305&m=dev) image which is more complete and not cut off from the right side. 
- the Core Site Banner Notifications from the server are also full width now.

**Comment by mohitwp** [comment]
### QA Update ⚠️ 

- Tested on main environment.
- Verified that Success notice banner is now width.
- Verified that core site banner notifications from server is now full width.


@jimmymadon
**Issue >** While testing the AdSense overlay notice, I observed that its alignment breaks and the position shifts when the screen width is between 1440px and 1499px.

<details>

<img width="2960" height="1356" alt="Image" src="https://github.com/user-attachments/assets/06145eb5-b2b8-487a-8cee-1114ce83e6a3" />

https://github.com/user-attachments/assets/4c71a696-69b7-4310-8dbf-66949823992e

</details>

**Q-2 )** -  I noticed that the AdSense overlay/notice appears slightly inside the content width boundary (highlighted in red).
Could you please check if this is expected behavior, or should the overlay be aligned more with the content container?

<details>

<img width="1880" height="694" alt="Image" src="https://github.com/user-attachments/assets/5de8f2f3-ab42-4b34-ad13-eea31b5e2a84" />

</details>


**PASS CASES**

<img width="3190" height="1254" alt="Image" src="https://github.com/user-attachments/assets/8b4a7893-c617-4357-b219-4881642da8a3" />

<img width="1914" height="778" alt="Image" src="https://github.com/user-attachments/assets/75c751e7-7890-482b-b56d-9c31180daf81" />

</details>

**Comment by jimmymadon** [comment]
@mohitwp Thanks and apologies - have created a new PR to fix issue 1. As mentioned on slack, Issue 2 is fine based on this [figma mock](https://www.figma.com/design/Jwd4oWQPtTJ4W9Y6JwMJjJ/Notifications?node-id=1398-12907&m=dev).

**Comment by mohitwp** [comment]
### QA Update ✅ 

- Tested on main environment
- Tested Splash screen, settings, main dashboard, entity dashboard and view only dashboard.
- Tested offline notice, overlay notification and surveys.
- Verified The splash, dashboard (including module setup flows), and settings screens should have a maximum width of 1366px for the main content area (that is to say, Site Kit's content excluding headers and footers).
- Verified The max-width content centered horizontally within the plugin's content area.
- Verified Setup CTA widgets and Notice type notifications (originally known as subtle notifications) constrained to the max-width as per this [Figma](https://www.figma.com/design/Jwd4oWQPtTJ4W9Y6JwMJjJ/Notifications?node-id=1150-49611&t=4LF9esNwXXeWauaB-0).
- Verified that Success notice banner is now width.
- Verified that core site banner notifications from server is now full width.
- Verified that overlay notice related issues are resolved now.

<details>

<img width="1546" height="617" alt="Image" src="https://github.com/user-attachments/assets/0d6bc453-343d-401b-86e4-d9df55b1afb8" />

https://github.com/user-attachments/assets/11934cd9-45e1-4b52-a201-cd4f7eb8ef28

<img width="3190" height="1254" alt="Image" src="https://github.com/user-attachments/assets/8b4a7893-c617-4357-b219-4881642da8a3" />

<img width="1914" height="778" alt="Image" src="https://github.com/user-attachments/assets/75c751e7-7890-482b-b56d-9c31180daf81" />

https://github.com/user-attachments/assets/573ae29a-d2f6-49f3-82bf-b3070c0c9e09

https://github.com/user-attachments/assets/2ad85b8e-be04-4250-9332-17b07ab972ea

https://github.com/user-attachments/assets/7e6f0d93-9fc6-467e-aa19-8c869d33b5ad

https://github.com/user-attachments/assets/e2d34ba0-cdef-498e-821f-9384039f8be1

<img width="3204" height="1278" alt="Image" src="https://github.com/user-attachments/assets/b5f2bb16-dec8-4c1a-b7c9-1d9f1557a69a" />

<img width="1917" height="862" alt="Image" src="https://github.com/user-attachments/assets/0da9e8b9-fd2a-4663-8e53-649e71638b26" />

<img width="3180" height="1274" alt="Image" src="https://github.com/user-attachments/assets/28e57e4c-2bd4-4aed-8de3-6a5987efec24" />

<img width="3214" height="1298" alt="Image" src="https://github.com/user-attachments/assets/4fe3dcf2-1b22-472f-9fb5-fe8f6501a97c" />

<img width="3180" height="1264" alt="Image" src="https://github.com/user-attachments/assets/4328a1d9-39af-49e5-ac71-c0efc1b31170" />

<img width="3212" height="1498" alt="Image" src="https://github.com/user-attachments/assets/22cb182a-7d07-4751-8a73-c4025efdfad5" />


<img width="3206" height="1256" alt="Image" src="https://github.com/user-attachments/assets/2010e6e6-3e1a-41eb-b6cd-3460faf6c5ea" />

<img width="1699" height="731" alt="Image" src="https://github.com/user-attachments/assets/efe8ba06-aaed-4bdc-be83-6bff515d393b" />

</details>
